pages/_lang/generate.vue

```diff
@@ -785,10 +814,11 @@ export default class extends Vue {
   @GeneratorGetters suggestionsTotal;
   @GeneratorGetters warningsTotal;
 
-  public created(): void {
+  public async created() {
```

Comment:
Why is this async? I don't see any await calls.

Reply:
Ahhh, that was from a fetch call I had in there earlier, let me remove that. Good find

Reply:
fixed https://github.com/pwa-builder/PWABuilder/pull/1065/commits/d572b373bf0cb5f9315371f00cf7c5dc6f71fa2f